pkg/apiserver/archive/archive.go

```diff
@@ -137,106 +123,71 @@ func (s *Service) list(c *gin.Context) {
 			Kind:      meta.Kind,
 			Namespace: meta.Namespace,
 			Name:      meta.Name,
-			CreatedAt: meta.StartTime,
+			Created:   meta.StartTime.Format(time.RFC3339),
 		})
 	}
 
 	c.JSON(http.StatusOK, archives)
 }
 
-// @Summary Get the detail of an archived chaos experiment.
-// @Description Get the detail of an archived chaos experiment.
+// @Summary Get a archived chaos experiment.
```

Comment:
"an archived xxxx"

Reply:
Yep, this and the above are two typos. I'll fix it right away. 🍺